libc/utils/gpu/loader/Loader.h

```diff
@@ -28,6 +28,7 @@ struct LaunchParameters {
   uint32_t num_blocks_x;
   uint32_t num_blocks_y;
   uint32_t num_blocks_z;
+  bool print_resource_usage;
```

Comment:
This isn't a launch parameter, it should just be a separate option. Honestly I'm wondering if I should just port this to use the LLVM commandline handling now that it's always built w/ LLVM.

Reply:
Ah, so you mean that checking resource usage shouldn't require launching a kernel, right?

Reply:
No, `launch_parameters` are specifically arguments to the kernel launch. The API expects stuff like the number of threads, blocks, dynamic memory, the stream, etc. This isn't related to that since the API doesn't use it.

Reply:
Ah, that makes sense. I can move resource usage logic out of the kernel launch logic

Reply:
Move this to a separate global and pass it to the function. Honestly I'm thinking I should just rewrite this to be a pure LLVM utility and just use their https://clang.llvm.org/docs/ClangCommandLineReference.html interface.